conn.go

```diff
@@ -347,6 +376,97 @@ func (conn *Conn) inWorker() {
 	}
 }
 
+// sigWorker runs in an own goroutine, getting signals and handling
```

Comment:
It is running in its own goroutine, in the same way like inWorker and outWorker - see addition in auth.go